pkg/apis/greenhouse/v1alpha1/pluginbundle_types.go

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PluginBundleSpec defines the desired state of PluginBundle
+type PluginBundleSpec struct {
+	// ClusterSelector is a label selector to select the clusters the plugin bundle should be deployed to.
+	ClusterSelector metav1.LabelSelector `json:"clusterSelector"`
+
+	// Plugins is a list of plugins that should be deployed.
+	Plugins []PluginConfigSpec `json:"plugins"`
```

Comment:
Re-use existing or have a new struct here?

Reply:
make additive only initially?

Reply:
On 2nd thought: This should only be a single spec 
```suggestion
	PluginConfigSpec PluginConfigSpec `json:"pluginConfigSpec"`
```

In other words, we'd have a `kube-monitoring` bundle that would ensure only this plugin in all selected clusters.